frontend/scripts/react-components/home/home.container.js

```diff
@@ -2,35 +2,25 @@ import { connect } from 'react-redux';
 import Home from 'react-components/home/home.component';
 import { BREAKPOINTS, HOME_VIDEO } from 'constants';
 import { setContextIsUserSelected, getTopCountries } from 'scripts/actions/app.actions';
-import {
-  playHomeVideo,
-  clickNextEntrypoint,
-  clickEntrypoint
-} from 'scripts/react-components/home/home.actions';
+import { playHomeVideo } from 'scripts/react-components/home/home.actions';
 import { getSelectedContext, getSelectedYears } from 'reducers/app.selectors';
 import { getDestinationCountries } from 'react-components/home/home.selectors';
+import sortBy from 'lodash/sortBy';
 
 function mapStateToProps(state) {
   const { query = {} } = state.location;
   const { tweets, posts, testimonials } = state.home;
   // not ideal, we could use redux-responsive if we need more changes like the one below
   const isSmallResolution = window.innerWidth <= BREAKPOINTS.small;
-  const INSIGHTS = ['INSIGHT', 'INFO BRIEF', 'ISSUE BRIEF', 'LONGER READ'];
   const promotedPost = posts.find(post => post.highlighted);
-  const blogPosts = posts.filter(
-    post => !(post.highlighted && !isSmallResolution) && !INSIGHTS.includes(post.category)
-  );
-  const insightsPosts = posts.filter(
-    post => !(post.highlighted && !isSmallResolution) && INSIGHTS.includes(post.category)
-  );
+  const insightsPosts = posts
+    .filter(post => !(post.highlighted && !isSmallResolution))
+    .concat(testimonials);
```

Comment:
testimonials are not showing up, but they're another thing. Could we remove this?